kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/spec/style/scopes/ContainerScope.kt

```diff
@@ -36,10 +36,25 @@ interface ContainerScope : CoroutineScope {
 
    val defaultConfig: TestCaseConfig
 
+   /**
+    * Registers a test to this scope, wrapping the test in the most appropriate scope.
+    * For example, invoking this method on a [DescribeSpec] will choose the `it` scope.
+    */
+   suspend fun addTest(
+      name: String,
+      test: suspend TestContext.() -> Unit,
+   ) {
+      TODO()
```

Comment:
should this delegate to another `addTest` - IIUC, this can fail at runtime.

Reply:
I tried to make it abstract, which shows that this is not implemented everywhere.

Reply:
AFAIK, if the test style does not support adding as a sub-test, it should be added to the current level.

Reply:
Fixed.